### PR TITLE
Update to Apex status - clarity on status

### PR DESCRIPTION
**Pull request by smaddox-sf**
- update the status of Apex to reflect that the CLI is updated
 - Add a little more detail on the setting so users understand the choice more clearly

### CODE REVIEW

**Comment by smaddox-sf** [comment]
@ekapner and @AnanyaJha - This is the doc update for the Apex CLI command update.  It's simply adding the checkmark to show the progress for Apex and adding a little detail to the settings.

***We should release this update close to when we release the new commands to the CLI.

File: docs/_articles/en/user-guide/perf-enhancements.md [hunk 1]
```diff
@@ -20,13 +20,13 @@ This architecture change is done through a phased approach.  For both the Apex l
...
 |-------------------------|:-----------------:|:---------------------------:|:-------------------------:|:------------:|
-| Apex Library	          |        ✔️          |               ✔️             |             ✔️             |              |
+| Apex Library	          |        ✔️          |               ✔️             |             ✔️             |       ✔️        |
 | Deploy Retrieve Library |        ✔️          |               ✔️             |             ✔️              |              |
 	
 
 ## Setup
 
-There is a VS Code setting for each library.  Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
+There is a VS Code setting for each library.  Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).  Setting to false means VS Code will call the CLI to execute those commands.
```
Comment (ekapner) [inline docs/_articles/en/user-guide/perf-enhancements.md:29]:
```suggestion
There is a VS Code setting for each library.  Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).  Setting to false means that VS Code uses the CLI to execute those commands. 
```

Comment (ekapner) [reply docs/_articles/en/user-guide/perf-enhancements.md:29]: @smaddox-sf - Does "uses the CLI" work?

**Review by ekapner — COMMENTED** [review]
a couple of minor tweaks

**Review by AnanyaJha — APPROVED** [review]

File: docs/_articles/ja/user-guide/perf-enhancements.md [hunk 2]
```diff
@@ -1,24 +1,39 @@
...
+
+|                         | Library complete	| Available to try in VS Code	| On by default in VS Code	| CLI Updated  |
+|-------------------------|:-----------------:|:---------------------------:|:-------------------------:|:------------:|
+| Apex Library	          |        ✔️          |               ✔️             |             ✔️             |       ✔️        |
+| Deploy Retrieve Library |        ✔️          |               ✔️             |             ✔️              |              |
+	
+
+## Setup
+
+There is a VS Code setting for each library.  Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).  Setting to false means that VS Code will call the CLI to execute those commands.
```
Comment (ekapner) [inline docs/_articles/ja/user-guide/perf-enhancements.md:29]: Make the same change here as above? Each library has a corresponding VS Code setting. Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS). Setting to false means that VS Code uses the CLI to execute those commands.

Comment (ekapner) [inline docs/_articles/ja/user-guide/perf-enhancements.md:29]:
```suggestion
Each library has a corresponding VS Code setting. Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS). Setting to false means that VS Code uses the CLI to execute those commands. 
```

Comment (ekapner) [reply docs/_articles/ja/user-guide/perf-enhancements.md:29]: I went ahead and suggested below to make it easier for you.

**Review by ekapner — APPROVED** [review]
Thanks for making the similar edit in the Japanese version.

File: docs/_articles/ja/user-guide/perf-enhancements.md (hunk 2 above)
Comment (AnanyaJha) [reply docs/_articles/ja/user-guide/perf-enhancements.md]: Thanks Emily! I added in the edit.

**Pull request merged**